packages/common/src/HardhatConfig.ts

```diff
@@ -108,32 +108,20 @@ export function getHardhatConfig(
       rinkeby: { chainId: 4, url: getNodeUrl("rinkeby", true, 4), accounts: { mnemonic } },
       goerli: { chainId: 5, url: getNodeUrl("goerli", true, 5), accounts: { mnemonic } },
       kovan: { chainId: 42, url: getNodeUrl("kovan", true, 42), accounts: { mnemonic } },
-      arbitrum: {
-        chainId: 42161,
-        url: getNodeUrl("arbitrum", true, 42161),
-        accounts: { mnemonic },
-        companionNetworks: { mainnet: "mainnet" },
-      },
-      "arbitrum-rinkeby": {
-        chainId: 421611,
-        url: getNodeUrl("arbitrum-rinkeby", true, 421611),
-        accounts: { mnemonic },
-      },
       optimism: {
         chainId: 10,
         url: getNodeUrl("optimism", true, 10),
         accounts: { mnemonic },
         companionNetworks: { mainnet: "mainnet" },
       },
       "optimism-kovan": { chainId: 69, url: getNodeUrl("optimism-kovan", true, 69), accounts: { mnemonic } },
-      "optimism-test": {
-        url: "http://127.0.0.1:8545",
-        accounts: { mnemonic: "test test test test test test test test test test test junk" },
-        // This sets the gas price to 0 for all transactions on L2. We do this because account balances are not yet
-        // automatically initiated with an ETH balance.
-        gasPrice: 0,
-        testWhitelist: ["oracle/Finder"],
-        testBlacklist,
+      xdai: {
+        chainId: 100,
+        url: getNodeUrl("xdai", true, 100),
+        gas: 500000,
```

Comment:
Is it important that we keep these gas and gasPrice defaults?

Reply:
You'll also want to add "xdai" as a companion network to mainnet

Reply:
ye these gas values are needed to actually send transactions on xdai. without them I was unable to send any transactions. I've added the companion network on main net.